### ISSUE TITLE
Parsoid header forwarding over HTTP or HTTPS?

### ISSUE THREAD

**Issue by borenstein** [milcord/docker-internal-mediawiki#1]
Hey there,

The VisualEditor/Parsoid [installation instructions](https://www.mediawiki.org/wiki/Extension:VisualEditor#Forwarding_Cookies_to_Parsoid) make a big deal about the idea that, without special configuration, cookie headers will be forwarded to Parsoid in plaintext over HTTP.

Ben Hutchins' Mediawiki container makes it easy to set up SSL. However, I'm not clear as to whether that would also cover the way that Mediawiki communicates with Parsoid. If I follow Ben's instructions for SSL while setting up this container, will I address the issue of Parsoid transmitting headers over HTTP? (I think not, but it's worth asking.)

I ask because I have not been able to figure out how to set up Parsoid to work over HTTPS, despite [some instructions on the topic](https://www.mediawiki.org/wiki/Extension:VisualEditor#Parsoid_over_HTTPS) on the setup page. I thought I might attempt to crowdsource the problem. :)

Best,
David

**Comment by benhutchins** [comment]
No, I haven't yet configured Parsoid to automatically work over SSL here. Definitely something I should do. Also need to do a round of updating for the dependencies.

**Comment by borenstein** [comment]
Any chance you know how to do it? If you can point me in the right
direction, I could attempt to make the fix and put in a pull request.

**Comment by benhutchins** [comment]
@biologicaldynamics Not without looking into it more, I haven't setup SSL with Parsoid yet so don't have that knowledge at this time.

**Comment by borenstein** [comment]
OK, thanks anyway! I appreciate your help and quick response. I'll let you
know if I figure it out.

## 

David Bruce Borenstein, PhD
781.710.2789 (m)
https://www.linkedin.com/in/davidborenstein

On Mon, Sep 26, 2016 at 3:42 PM, Benjamin Hutchins <notifications@github.com
